hera_qm/xrfi.py

```diff
@@ -299,4 +301,51 @@ def xrfi_run(files, opts, history):
         else:
             raise ValueError('Unrecognized output file format ' + str(opts.outfile_format))
 
+        if opts.summary:
+            sum_file = ''.join([basename, opts.summary_ext])
+            sum_path = os.path.join(dirname, sum_file)
+            summarize_flags(uvd, sum_path)
     return
+
+
+def summarize_flags(uv, outfile):
+    """ Collapse several dimensions of a UVData flag array to summarize.
+    Args:
+        uv -- UVData object containing flag_array to be summarized
+        outfile -- filename for output npz file
+
+    Return:
+        npz file containing "spectrum" of max, min, mean, std, median of flags
```

Comment:
Can we add some documentation of what's in these files? Maybe we should explicitly state what keys are present and a quick description. It would be helpful for reading them back in later.